### ISSUE TITLE
Hinzufügen eines DevConfig-Schaltflächensymbols

### ISSUE THREAD

**Issue by andrzejgilas** [jens-maus/RaspberryMatic#2811]
### Describe the solution you'd like

Hinzufügen eines DevConfig-Schaltflächensymbols

genau wie auf diesem Bild: 
![3](https://github.com/user-attachments/assets/8499e39b-8591-4236-b666-a6dbd50d6ffb)

### Describe alternatives you've considered

keine

### Is your feature request related to a problem?

keine

### Additional information

[https://homematic-forum.de/forum/viewtopic.php?p=374579#p374579](url)

![1](https://github.com/user-attachments/assets/462c4f5b-30bf-45d5-a332-90bee6131b7f)

einfach zur Datei hinzufügen: 
`<img id=\"showCouplingCPHelp\" src=\"/ise/img/help.png\">`
In der Datei /www/config/control_panel.cgi

vor: 
`if { "[read_var /etc/config/tweaks CP_DEVCONFIG]" != "" } {
  puts "<div class=\"cpCell\"><div class=\"cpButton\">"
  puts "<div class=\"StdTableBtn CLASS21701\" onclick=\"window.open('/tools/devconfig.cgi?sid=$sid');\">DevConfig</div>"
  puts "<div class=\"StdTableBtnHelp\"></div></div></div>"
}`

nacher:
`if { "[read_var /etc/config/tweaks CP_DEVCONFIG]" != "" } {
  puts "<div class=\"cpCell\"><div class=\"cpButton\">"
  puts "<div class=\"StdTableBtn CLASS21701\" onclick=\"window.open('/tools/devconfig.cgi?sid=$sid');\">DevConfig</div>"
  puts "<div class=\"StdTableBtnHelp\"><img id=\"showCouplingCPHelp\" src=\"/ise/img/help.png\"></div></div></div>"
}`

**Comment by Baxxy13** [comment]
Kosmetik... 
Wir sollten uns dann aber auch einen aussagekräftigen Text einfallen lassen der angezeigt wird wenn der Mauspfeil über dem Button ist.

**Comment by Maik2208** [comment]
> Kosmetik... 
Stimmt. Aber auch wieder erschreckend dass diese Inkonsequenz dem ursprünglichen Entwickler scheinbar völlig egal ist...

**Comment by MichaelN0815** [comment]
Stimmt. Das ist meiner Meinung nach einer der wichtigsten PR diesen Jahres.

**Comment by andrzejgilas** [comment]
@MichaelN0815  Danke für deinen Kommentar. Ohne Menschen wie Sie wären wir im Chaos.
